docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1272,13 +1272,21 @@
 
                 <tr>
                     <td class="property" id="store-changelog-replication-factor">stores.<span class="store">store-name</span>.changelog.<br>replication.factor</td>
-                    <td class="default">2</td>
+                    <td class="default">stores.changelog.replication.factor.default</td>
```

Comment:
No problem on the late reply.  This is a pretty low priority request.

You suggestion works for me if there is a broad consensus that the risk that someone currently uses a a store name of "default" is acceptably low.  We certainly have no stores with that name.

Not sure how to document "default" as a reserved word.  I will look around, but, any pointers would be appreciated.